### ISSUE TITLE
Sorry, I had trouble doing what you asked.  Please try again...

### ISSUE THREAD

**Issue by PaPpY55** [spenwall/myq-alexa#12]
This code has been solid for me until recently where if I go to open or close the garage door, I get the response: "Sorry, I had trouble doing what you asked. Please try again."  When I ask for the status, it tells me "The door is undefined".  Is this happening for others as well and if so, any thoughts on a fix?  Thanks!

**Comment by N3rd7** [comment]
Yes I just tried and have the same problem! It has worked perfectly until
now

**Comment by jincollingo** [comment]
I had been using this code for a while now without issues.  I also use the Home Assistant MyQ integration to display the Garage Door information on a wall mounted tablet.  That integration also broke.  There was a work around to fix it on Home Assistant so I assume the code here would need to be modified.  The error stems from the USER AGENT.  So, Chamberlain changed things on their end that broke a lot of integrations from 3rd party creators.

**Comment by spenwall** [comment]
I will have a look at this soon. Thanks for the heads up

**Comment by goku1327** [comment]
Same issue. Thanks for looking into it.

**Comment by tschommer** [comment]
Same issue, I'm so annoyed by it.  Rooting for you spenwall, u r my saving grace for my garage door sanity.  Hoping for an update soon!  🙏  Thank you!

**Comment by spenwall** [comment]
Ok bad news. I just came across this article about MyQ cutting off any unauthorized 3rd party API access. https://www.theverge.com/23949612/chamberlain-myq-smart-garage-door-controller-homebridge-integrations

This really sucks as I just bought a new controller. Hoping now that I can send it back. I will keep an eye on things and see if we can find a way around this but it isn't looking good.

**Comment by PaPpY55** [comment]
Appreciate the update Spencer!  Sad news indeed!

**Comment by dreh65** [comment]
We knew this was coming. MyQ has been a greedy, good for nothing company
for some time. Unfortunately, I’m in the same boat where my (soon to be
former) garage man sold me the very expensive and now mostly useless
integrated unit instead of a plain old opener that I could have added my
own smart device to.

This is the third garage opener I’ve had in 23 years, so I’ll wait for this
one to go, then just get something new and hopefully better.

Donny Hornstein

**Comment by goku1327** [comment]
That is super lame. 
I was able to get it working with hubitat, but I need to tell alexa the pin every time I open the door.

**Comment by jincollingo** [comment]
I had moved away from the Amazon developer platform and put the MyQ integration onto Home Assistant. It worked quite well for a while but started having issues as a result of Chamberlain changing the API.  So I spun up HomeKit on one of my Synology's and that has been working reliably ever since.
